00-EssentialShortcuts/0.2-GoToEverything.md

```diff
@@ -1,9 +1,8 @@
-﻿# Essential Shortcuts - Go To Everything
+﻿# Essential Shortcuts - Search Everywhere
 
-Navigate anywhere by name combines go to type, symbol and file in one menu.
+Search everywhere by name combines go to type, symbol, actions and file in one menu.
 
-> Ctrl+T (VS)
-> Ctrl+N (IntelliJ)
+<shortcut id="Search Everywhere">`Ctrl+T`</shortcut>
```

Comment:
Any reason for the backticks here? Would keep this clean for eventual automation :-)